mail_embed_image/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.mail</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="mail.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <div id="companies_setting" position="inside">
+                <br />
+                <div
+                    class="o_setting_right_pane"
+                    id="mail_templates_setting"
+                    groups="mail.group_mail_template_editor,base.group_system"
+                >
+                    <span class="o_form_label">Email Preprocessing</span>
+                    <div class="text-muted">
+                        Method used to embed images in HTML emails. CIDs attachment does not work with all email clients. Data SRC is more reliable.
```

Comment:
that actually depends very much on the clients you target, with old outlooks it's the other way around. I'd propose not to make claim about what works better, and rather suggest to test with whatever are the expected mail clients